model/Core/Vocabularies/RelationshipType.md

```diff
@@ -66,6 +66,16 @@ Build Profile specific RelationshipType descriptions can be found [here](https:/
 - buildInvokedBy: Agent that invoked the build
 - buildOnBehalfOf: Actor for which buildInvokedBy is acting on behalf of
 - buildHostOf: Element in which the build instance runs on
+- advisory: (Security) Used to associate a security vulnerability with an element
+- coordinatedBy: (Security) TODOdescription
+- exploitCreatedBy: (Security) TODOdescription
+- fixedBy: (Security) Designates a vulnerability has been fixed by an element, actor or entity
```

Comment:
`element, actor or entity` ? Looks like the first item should probably not be `element`?

Reply:
If it is intended to be used for `fixedBy $whom` and `fixedBy $what`, sounds like overloading which is not good. This are two semantically different statements and they should be encoded with different types